### PR TITLE
Publication card scrolling issue

### PR DESCRIPTION
**Pull request by neelesh-tippe**
Publication card scrolling issue

### CODE REVIEW

**Review by shishirphadke — CHANGES_REQUESTED** [review]

File: stories/Molecules/Blocks/PublicationCard/publicationcard.scss [hunk 1]
```diff
@@ -81,10 +79,25 @@
...
+      .external-link {
+        margin-left: 0;
+        margin-right: $spacing-12;
+      }
+    }
+
+    &__description {
+      .grid-x>:not(:first-of-type) {
+        border-left: 0;
+        border-right: $spacing-02 solid #ced0d3;
```
Comment (shishirphadke) [inline stories/Molecules/Blocks/PublicationCard/publicationcard.scss:94]: @neelesh-tippe Kindly do not use hardcoded color values.

File: stories/assets/js/sticky.js [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
+      var generalSidebarHeight = $sticky.innerHeight();
+      var stickyTop = $sticky.offset().top;
+      var stickOffset = 0;
+      var stickyStopperPosition = $stickyrStopper.offset().top;
+      var stopPoint = stickyStopperPosition -(generalSidebarHeight +stickOffset);
+      var diff = stopPoint -($('footer').innerHeight()-150);
+      
+      $(window).scroll(function(){
+        var windowTop = $(window).scrollTop();
+        if($(window).width() > 834){
```
Comment (shishirphadke) [inline stories/assets/js/sticky.js:14]: @neelesh-tippe kindly refrain from using hardcoded values. Kindly use a global config file to define viewport values.

File: stories/Molecules/Blocks/PublicationCard/publicationcard.scss (hunk 1 above)
Comment (neelesh-tippe) [reply stories/Molecules/Blocks/PublicationCard/publicationcard.scss:94]: there is no variable for above color code.

File: stories/assets/js/variables.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const BREAKPOINT = window.BREAKPOINT || {};
```
Comment (shishirphadke) [inline stories/assets/js/variables.js:1]: @neelesh-tippe Kindly change this to something more robust. 
Eg.
Change the filename to something like `global-variables.js`. And call the variables something like this:
```
// In global-variables.js

// Define functions which can be called globally.
const foo = (x) => { 
  return x; 
}
const bar = (y) => {
  return y;
}
const baz = (z) => {
  return z;
}

// Define Global Breakpoints in Js.
const breakpoints = {
  small: 767,
  medium: 768,
  large: 1024,
  extraLarge: 1440,
  mediumTab: 1439,
  tablet: 834,
  ...
};

export {foo, bar, baz, breakpoints};
----

// In your Js file.
import { foo, breakpoints } from './global-variables';
...
...
...
console.log(foo(5)); // 5
console.log(breakpoints.small); // 767
console.log(breakpoints.medium); // 768
```

**Pull request closed without merging**